src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/FileStorageService.cs

```diff
@@ -65,12 +62,9 @@ public async Task<Stream> StreamBlob(BlobInfo blob)
             return await _blobStorageService.StreamBlob(PrivateFilesContainerName, blob.Path);
         }
 
-        public async Task DeleteBatchFile(Guid releaseId, string dataFileName)
+        public async Task DeleteBlobByPath(string blobPath)
         {
-            await _blobStorageService.DeleteBlob(
-                PrivateFilesContainerName,
-                AdminReleaseDirectoryPath(releaseId, ReleaseFileTypes.Data) + dataFileName
```

Comment:
You can use `AdminReleasePath`

Reply:
Done, thanks (not here though as these lines are being deleted)